modules/sdk-coin-avaxp/src/lib/utils.ts

```diff
@@ -192,27 +192,26 @@ export class Utils implements BaseUtils {
     network: AvalancheNetwork,
     message: BufferAvax,
     signature: BufferAvax,
-    prv: BufferAvax
+    pubkey: BufferAvax
   ): boolean {
     const ky = new KeyPairAvax(network.hrp, network.networkID.toString());
-    ky.importKey(prv);
-    return ky.verify(message, signature);
+    return ky.recover(message, signature).equals(pubkey);
   }
 
   /**
    * Avaxp wrapper to verify signature
    * @param network
    * @param message
    * @param signature
-   * @param prv
+   * @param pubkey
    * @return true if it's verify successful
    */
-  verifySignature(network: AvalancheNetwork, message: Buffer, signature: Buffer, prv: Buffer): boolean {
+  verifySignature(network: AvalancheNetwork, message: Buffer, signature: Buffer, pubkey: Buffer): boolean {
```

Comment:
Missing breaking change documentation in commit.

Reply:
Surely, if the method is used... but since the verification in WP is done with the public key, this method is not being used anywhere. However, if you still request, I'll add the breaking changes doc.

Reply:
Yes, due to this being an open source repository, these must be recorded as breaking changes for our end users.